### PR TITLE
py3-ml-metadata: new package

### PR DESCRIPTION
**Pull request by kaniini**
It took 2 of our greatest engineers, but we finally made this one happen.

### CODE REVIEW

File: py3-ml-metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package:
+  name: py3-ml-metadata
+  version: 1.14.0
+  epoch: 0
+  description: "foo"
```
Comment (imjasonh) [inline py3-ml-metadata.yaml:5]:
```suggestion
  description: For recording and retrieving metadata associated with ML developer and data scientist workflows.
```

**Review by imjasonh — CHANGES_REQUESTED** [review]

File: py3-ml-metadata/0002-add-zetasql-hackfix.patch [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+@@ -249,13 +249,13 @@ http_archive(
+ )
+ 
+ # BEGIN IFNDEF_WIN
+-ZETASQL_COMMIT = "ac37cf5c0d80b5605176fc0f29e87b12f00be693" # 08/10/2022  # windows
++ZETASQL_COMMIT = "c9fc70e7b4b144eb59e0e5f138353592eb8b9199" # 08/10/2022  # windows
+ http_archive(  # windows
+     name = "com_google_zetasql",  # windows
+-    urls = ["https://github.com/google/zetasql/archive/%s.zip" % ZETASQL_COMMIT],  # windows
++    urls = ["https://github.com/kaniini/zetasql/archive/%s.zip" % ZETASQL_COMMIT],  # windows
```
Comment (imjasonh) [inline py3-ml-metadata/0002-add-zetasql-hackfix.patch:24]: ditto

Comment (kaniini) [reply py3-ml-metadata/0002-add-zetasql-hackfix.patch:24]: ditto

File: py3-ml-metadata/0001-WORKSPACE-bump-abseil-version.patch [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+--- a/WORKSPACE
++++ b/WORKSPACE
+@@ -43,9 +43,9 @@ rules_foreign_cc_dependencies()
+ 
+ http_archive(
+     name = "com_google_absl",
+-    urls = ["https://github.com/abseil/abseil-cpp/archive/940c06c25d2953f44310b68eb8aab6114dba11fb.zip"],
+-    strip_prefix = "abseil-cpp-940c06c25d2953f44310b68eb8aab6114dba11fb",
+-    sha256 = "0e800799aa64d0b4d354f3ff317bbd5fbf42f3a522ab0456bb749fc8d3b67415",
++    urls = ["https://github.com/kaniini/abseil-cpp/archive/0f320df2d86fd76cdeb57709f4579c8bc4ecb606.zip"],
```
[the remark this answers is not in the document]
Comment (kaniini) [reply py3-ml-metadata/0001-WORKSPACE-bump-abseil-version.patch]: I wound up solving this by switching to inline patching.

**Review by imjasonh — APPROVED** [review]

**Pull request merged**